docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+# Monitor Kubernetes 
+
+Monitoring the state of the database is crucial to timely identify and react to performance issues. [Percona Monitoring and Management (PMM) solution enables you to do just that](monitoring.md).
+
+However, the database state also depends on the state of the Kubernetes cluster itself. Hence it’s important to have metrics that can depict the state of the Kubernetes cluster.
+
+This document describes how to set up monitoring of the Kubernetes cluster health. This setup has been tested with the [PMM server](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html#pmm-server) as the centralized data storage and the Victoria Metrics operator as the statistics collector, though the steps should also apply to other monitoring applications. 
+
+## Considerations
+
+1. In this setup we use [Victoria Metrics kubernetes monitoring stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack) Helm chart. When customizing the chart's values, consider the following:
+
+    * Since data is stored in the PMM server, disable VMSingle and VMCluster.
+    * The Prometheus node exporter is not installed by default since it requires  privileged containers that need to access the host file system. If you do need the node exporter, set the `prometheus-node-exporter.  enabled` flag in the `values.yaml` file to `true`.
+    * Check all the role-based access control rules provided by the charts and modify them based on your requirements. 
+
+2. This setup is used for a 1:1 mapping from Kubernetes cluster to the PMM server. If you wish to monitor more than one Kubernetes cluster in a single PMM server, you need to configure the dashboard to filter a specific Kubernetes cluster. You also need to properly relabel the metrics and from the backend. 
```

Comment:
```suggestion
2. This setup is used for a 1:1 mapping from Kubernetes cluster to the PMM Server. If you wish to monitor more than one Kubernetes cluster in a single PMM Server, you need to configure the dashboard to filter a specific Kubernetes cluster. You also need to properly relabel the metrics and from the backend. 
```

Once again, it would be nice to have here some link to the external doc... send user to Grafana documentation about custom dashboards, something like this